spec/serializers/topic_view_serializer_spec.rb

```diff
@@ -64,4 +66,44 @@
       end
     end
   end
+
+  let(:user) { Fabricate(:user) }
+  let(:moderator) { Fabricate(:moderator) }
```

Comment:
I think we should declare "global" `let`s at the top of the file to avoid duplications. `let` specific to a particular test case should probably be declared within the scope of the `describe` block.